products/big-ip.md

```diff
@@ -0,0 +1,44 @@
+---
+title: BIG-IP 
+category: os
+permalink: /big-ip
+versionCommand: show /sys version
+releaseDateColumn: true
+eolColumn: End of Technical Support
+eoasColumn: End of Software Development
+
+releases:
+
+-   releaseCycle: "17.1.x"
+    releaseDate: 2023-03-14
+    eoas: 2027-03-31
+    eol: 2027-03-31
+    latest: "17.1.1"  
+    lts: true
+
+-   releaseCycle: "16.1.x"
+    releaseDate: 2021-07-07
+    eoas: 2025-07-31
+    eol: 2025-07-31
+    latest: "16.1.5"
+    lts: true
+
+-   releaseCycle: "15.1.x"
+    releaseDate: 2019-12-11
+    eoas: 2025-12-31
+    eol: 2025-12-31
+    latest: "15.1.10"
+    lts: true
+
```

Comment:
It would be nice to document a few (let's say 2 or 3) unsupported releases too (as visible on https://my.f5.com/manage/s/article/K5903).

Reply:
I am missing release date of old release so currently build failed I'll revert back 15.1 is EOL  so we have one EOL release at the moment is this enough ?

Reply:
Yes, one is enough.

Reply:
Added some information based on https://my.f5.com/manage/s/article/K9412.